<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recognition</class>
 <widget class="QWidget" name="Recognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="cbox_type">
        <item>
         <property name="text">
          <string>GeometricConsistencyGrouping</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hough3DGrouping</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="spin_gcthres">
            <property name="prefix">
             <string>GCThreshold: </string>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dspin_gcsize">
            <property name="prefix">
             <string>GCSize： </string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QDoubleSpinBox" name="dspin_binsize">
            <property name="prefix">
             <string>HoughBinSize: </string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dspin_houghthres">
            <property name="prefix">
             <string>HoughThreshold: </string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
            <property name="value">
             <double>-1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="check_useinter">
            <property name="text">
             <string>UseInterpolation</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="check_usedis">
            <property name="text">
             <string>UseDistanceWeight</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5,5">
        <item>
         <widget class="QCheckBox" name="check_keypoints">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/snap-bounding-box-center.svg</normaloff>:/icon/resource/icon/snap-bounding-box-center.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_model">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>SetModel</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/source.svg</normaloff>:/icon/resource/icon/source.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_scene">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>SetScene</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/target.svg</normaloff>:/icon/resource/icon/target.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_preview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/preview.svg</normaloff>:/icon/resource/icon/preview.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_add">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/add.svg</normaloff>:/icon/resource/icon/add.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_apply">
          <property name="text">
           <string>Apply</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/apply1.svg</normaloff>:/icon/resource/icon/apply1.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_reset">
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset resource="../../src.qrc">
            <normaloff>:/icon/resource/icon/reset.svg</normaloff>:/icon/resource/icon/reset.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbox_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
